config/application.rb

```diff
@@ -55,7 +55,7 @@
 module Mastodon
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 6.1
+    config.load_defaults 7.0
```

Comment:
It may be a good idea to cut a release before bumping this value. The `7.0` defaults includes a change to `active_support.cache_format`:
https://github.com/rails/rails/blob/4408b67742cb7ee3cdc3b3d850e1a3be205bd4e6/railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_0.rb.tt#L108

The safe way to update the cache format is to deploy everything on Rails 7 with the old cache format, and then only changing the format once everything is on Rails 7.

Reply:
:+1: Yep, that makes sense to me